apps/meteor/client/views/admin/settings/MemoizedSetting.tsx

```diff
@@ -18,23 +19,39 @@ import SelectSettingInput from './inputs/SelectSettingInput';
 import SelectTimezoneSettingInput from './inputs/SelectTimezoneSettingInput';
 import StringSettingInput from './inputs/StringSettingInput';
 
+type MemoizedSettingProps = {
+	type: ISettingBase['type'];
+	hint?: ReactElement;
+	callout?: ReactElement;
+	value?: SettingValue;
+	editor?: SettingEditor;
+	onChangeValue?: (value: unknown) => void;
+	onChangeEditor?: (value: unknown) => void;
+	onResetButtonClick?: () => void;
+	className?: string;
+	invisible?: boolean;
+	label?: string;
+	sectionChanged?: boolean;
+	hasResetButton?: boolean;
+};
+
 const MemoizedSetting = ({
 	type,
 	hint = undefined,
 	callout = undefined,
 	value = undefined,
 	editor = undefined,
-	onChangeValue = () => {},
-	onChangeEditor = () => {},
+	onChangeValue,
+	onChangeEditor,
 	className = undefined,
 	invisible = undefined,
 	...inputProps
-}) => {
+}: MemoizedSettingProps): ReactElement => {
 	if (invisible) {
-		return null;
+		return <></>;
 	}
 
-	const InputComponent =
+	const InputComponent: FC<any> =
```

Comment:
Don't use FC anymore, will be deprecated in React soon, just type the props properly